i18n/create_msg.js

```diff
@@ -63,6 +77,19 @@ Blockly.CatblocksMsgs.getCurrentLocaleValues = function() {
   return Blockly.CatblocksMsgs.locales[Blockly.CatblocksMsgs.getCurrentLocale()];
 };
 
+Blockly.CatblocksMsgs.loadNewLocale = function(locale) {
+  const scripts = document.getElementsByTagName("script");
+  const baseURI = scripts[0].baseURI;
+  let json_object = [];
+  const url = baseURI + "//i18n//" + locale + ".json";
```

Comment:
Request is send to host///i18n//name.json, please fix

Reply:
fixed it with your suggestion below.